package.json

```diff
@@ -45,7 +45,7 @@
     "yeoman-test": "^1.4.0"
   },
   "engines": {
-    "node": ">= 5.0.0"
+    "node": ">= 6.0.0"
```

Comment:
You probably don't want to ignore the `lib/` folder. https://github.com/feathersjs/generator-feathers-plugin/blob/12f0db84a89f30d1a00c09e0b0664ae3a9172e3a/package.json#L36